packages/babel-traverse/src/path/evaluation.js

```diff
@@ -112,25 +112,19 @@ export function evaluate(): { confident: boolean; value: any } {
     }
 
     if (path.isTemplateLiteral()) {
-      let str = "";
-
-      let i = 0;
-      const exprs = path.get("expressions");
-
-      for (const elem of (node.quasis: Array<Object>)) {
-        // not confident, evaluated an expression we don't like
-        if (!confident) break;
+      return evaluateQuasis(path, node.quasis);
+    }
 
-        // add on cooked element
-        str += elem.value.cooked;
+    if (path.isTaggedTemplateExpression()) {
```

Comment:
Yeah it's necessary since if it's an identifier then you would be doing `const { node: { name } } = object;` of undefined since `const object = path.get("tag.object");` object is undefined in that case